docs/examples.md

```diff
@@ -0,0 +1,99 @@
+
+# Textbook Examples
+
+A significant examples directory attached to cirq contains many useful
+examples that you can use to see how to write different algorithms.
+Most of these can be found in the
+[examples directory](https://github.com/quantumlib/Cirq/tree/master/examples)
+in the Github repository.
+
+In addition to these examples, you can also find the notebooks that generated
+the cirq docs found here in the
+[docs directory](https://github.com/quantumlib/Cirq/tree/master/docs) as well
+as all the more advanced case studies listed below this page in the table of
+contents.
+
+Below is a listing of the examples in the repository:
+
+## Beginning algorithms
+
+*    [Hello Qubit](https://github.com/quantumlib/Cirq/blob/master/examples/hello_qubit.py)
+Simple first program showing how to create a circuit.
+
+*    [Deutsch Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/deutsch.py)
+Textbook example of the simplest quantum advantage.
+
+*    [Bernstein-Vazirani](https://github.com/quantumlib/Cirq/blob/master/examples/bernstein_vazirani.py)
+Textbook algorithm determining properties of a hidden "oracle" function
+
+*    [Bell Inequality](https://github.com/quantumlib/Cirq/blob/master/examples/bell_inequality.py)
+Demonstration of a Bell inequality which shows violation of local hidden variables
+
+*    [BB84](https://github.com/quantumlib/Cirq/blob/master/examples/bb84.py)
+Textbook algorithm for Quantum key distribution
+
+*    [Noisy simulation](https://github.com/quantumlib/Cirq/blob/master/examples/noisy_simulation_example.py)
+     How to use a noisy simulator to generate results with amplitiude damping.
+
+*    [Line placement](https://github.com/quantumlib/Cirq/blob/master/examples/place_on_bristlecone.py)
+How to find a line of adjacent qubits on a device
+
+*    [Quantum Teleportation](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_teleportation.py)
+A demonstration of using 2 classical bits to transport a quantum state from one
+qubit to another.
+
+*    [Super dense coding](https://github.com/quantumlib/Cirq/blob/master/examples/superdense_coding.py)
+     Transmit 2 classical bits using one quantum bit.
+
+
+# Intermediate Textbook Algorithms
+
+*    [Grover Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/grover.py)
+Example of applying the textbook Grover algorithm to find a 2-bit oracle
+function.
```

Comment:
We're not finding the oracle function. We're finding an element. The function (which is accessed as an oracle, i.e. a black box) merely serves as a predicate that decides what it is that we're looking for.

Suggestion: "Textbook algorithm for finding a needle in a haystack."

(The suggestion also removes the name of the algorithm which is repeated - see link on line above - and unnecessary "Example of applying").

Reply:
I didn't particularly like your suggestion.  How's this instead:
"Textbook algorithm for finding a single element hidden within a oracle function"